site/en/docs/privacy-sandbox/attribution-reporting/understanding-noise/index.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Understanding noise'
+subhead: >
+  What noise means, where it is added, and how it impacts your measurement efforts.
+description: >
+  What noise means, where it is added, and how it impacts your measurement efforts.
+date: 2022-06-23
+updated: 2022-06-23
+authors:
+  - maudn
+---
+
+Data in aggregatable reports is _noised_; in other words, for a small percentage of cases, random data is sent instead of real reports/data. This practice preserves user privacy by preventing the joining of user identity across sites. 
+
```

Comment:
```suggestion
Summary reports are the result of the aggregation of _aggregatable reports_. 
When aggregatable reports are batched by a collector and processed by the aggregation service, noise—a random amount of data—is added to the resulting summary reports. 
Noise is added to protect user privacy. The goal of this mechanism is to have a framework which can support [differentially private](https://github.com/WICG/conversion-measurement-api/blob/main/AGGREGATE.md#differential-privacy) measurement.

```

* The current text is more accurate for event-level reports - Suggestion to replace it with text that's more accurate for aggregatable / summary reports
* Could we consider adding this diagram?
https://docs.google.com/document/d/1bU0a_njpDcRd9vDR0AJjwJjrf3Or8vAzyfuK8JZDEfo/edit#bookmark=id.kadbfacjv8de

Reply:
Adding. thanks!